### PR TITLE
chore: upgrade openapi dependency to feat/r17

### PR DESCRIPTION
**Pull request by ian60509**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -63,6 +70,7 @@ require (
 	golang.org/x/text v0.14.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/protobuf v1.33.0 // indirect
-	gopkg.in/h2non/gock.v1 v1.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/free5gc/openapi => github.com/yccodr/openapi v1.0.9-0.20240801032959-f8c907cee3a4
```
Comment (andy89923) [inline go.mod:76]: directly use free5gc/openapi hash

**Comment by andy89923** [comment]
Is the PR ready?

**Comment by donald1218** [comment]
LGTM!

**Comment by andy89923** [comment]
Add `GroupIDmap` & `ImsDataRepository` service api with not implement response.

File: internal/sbi/processor/influence_data.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func (p *Processor) ApplicationDataInfluenceDataGetProcedure(c *gin.Context, col
 
 func (p *Processor) ParseSnssaisFromQueryParam(snssaiStr string) []models.Snssai {
 	var snssais []models.Snssai
-	err := json.Unmarshal([]byte("["+snssaiStr+"]"), &snssais)
+	err := json.Unmarshal([]byte(snssaiStr), &snssais)
```
Comment (ianchen0119) [inline internal/sbi/processor/influence_data.go:51]: @ian60509 Why does this line need to be changed?

**Comment by ianchen0119** [comment]
Overall, it looks good to me.

File: internal/sbi/processor/influence_data.go (hunk 2 above)
Comment (ian60509) [reply internal/sbi/processor/influence_data.go:51]: According to TS 29.571 Snssai definition : 
* pattern: `'^[A-Fa-f0-9]{6}$'`
I revised it to comply with the spec

**Review by ianchen0119 — APPROVED** [review]

**Pull request merged**